src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/RunWorkflowTests.kt

```diff
@@ -212,6 +215,8 @@ class RunWorkflowTests : SeleniumTest()
         addButton.click()
 
         wait.until(ExpectedConditions.presenceOfElementLocated(By.id("workflow-report-0")))
+        val xButton = driver.findElement(By.cssSelector(".btn-outline-secondary"))
+        xButton.click()
```

Comment:
This clears the report from the input allowing the user to add additional reports. Mark has suggested that the input should be cleared automatically in the ui.